### DISCUSSION TITLE
通知・お知らせ（一般）

### DISCUSSION THREAD

**Discussion by r-saito-cw** [Personal-Data-Linkage-Module/pxr-ver-1.0#107] [告知]
開発貢献やツールの公開など開発貢献者から周知したい内容がある際は、こちらに投稿ください。

**Comment by m-ukai0** [comment]
開発貢献活動として以下Pull Requestいたしました。（https://github.com/Personal-Data-Linkage-Module/pxr-linkage/pull/108）

環境構築をAWSで実施した際、リソース作成に使用したCFnテンプレートをアップさせていただきます。
今後AWSで構築する際の参考資料としてご活用ください。

■今回追加したファイル
　pxr-linkage/tool/
　　∟CFn_pxr_pf_templete.yaml　⇒　構築に必要なAWSリソースを作成
　　∟CFn_pxr_role_templete.yaml　⇒　構築に必要なIAMロールを作成
　　∟CFn_pxr_monitoring_pf.yaml　⇒　監視系のリソースを作成
　　∟Cfnテンプレート_設定値一覧_sample.xlsx　⇒　上記CFnテンプレート実行に必要な設定値

**Reply by r-saito-cw** [reply]
ありがとうございます。皆様に広く使いいただけることを期待しています。
